app/views/recipes/_form.html.haml

```diff
@@ -8,6 +8,9 @@
   - Recipe::TIME_ATTRIBUTES.each do |time_attribute|
     = f.input time_attribute, label: "#{Recipe.human_attribute_name time_attribute} (in Minutes)"
   
+  %h2 Notes
+  = f.input :notes, label: false
+  
   %h2 Ingredients
   #ingredients
```

Comment:
Which comment?   The #ingredients line?  That's haml syntax for <div id="ingredients">...

Those div IDs are actually not needed because we won't want to style the input fields for ingredients any differently from those of  steps etc..  Because of the headings they also do not really help with the page structure.

We should either remove them everywhere or add them everywhere.  I'm OK with removing them.